scene/resources/sprite_frames.cpp

```diff
@@ -106,6 +106,12 @@ bool SpriteFrames::has_animation(const StringName &p_anim) const {
 	return animations.has(p_anim);
 }
 
+void SpriteFrames::duplicate_animation(const StringName &p_from, const StringName &p_to) {
+	ERR_FAIL_COND_MSG(!animations.has(p_from), "SpriteFrames doesn't have animation '" + String(p_from) + "'.");
+	ERR_FAIL_COND_MSG(animations.has(p_to), "Animation '" + String(p_to) + "' already exists.");
+	animations[p_to] = animations[p_from];
```

Comment:
I was just about to send a review talking about the flaw but you were faster than words can be written.

Use `vformat()`here. Not sure if the error messages could be worded better.
```suggestion
	ERR_FAIL_COND_MSG(!animations.has(p_from), vformat("SpriteFrames doesn't have animation '%s'.", p_from));
	ERR_FAIL_COND_MSG(animations.has(p_to), vformat("Animation '%s' already exists.", p_to));
	animations[p_to] = animations[p_from];
```